packages/css/src/button.css

```diff
@@ -42,6 +42,7 @@
   }
 
   align-items: center;
+  align-self: center; /* If placed in flex container */
```

Comment:
I don't think we should assume our button should always be in the center when in a flex container 🤔

Reply:
A button will always stretch to the same height as other items when placed in flex - so there will be a default regardless of what we do. My experience is that vertical centering relative to siblings is mostly what you want to achieve instead of siblings changing the button height.
<img width="350" alt="image" src="https://github.com/user-attachments/assets/22588298-5b85-4622-af60-7340b47ad935" />
vs. 
<img width="345" alt="image" src="https://github.com/user-attachments/assets/e77901f5-0fd8-40ae-bbe4-4cea5e906376" />

Reply:
hm, what do you think about `height: fit-content` instead?

Reply:
oh, that can work - good point! I'll do that :)

Reply:
Awesome, can you add a separate changeste for the button as well?